resources/scripts/components/server/schedules/EditScheduleModal.tsx

```diff
@@ -99,6 +102,20 @@ const EditScheduleModal = ({ schedule }: Props) => {
                         <Field name={'month'} label={'Month'}/>
                         <Field name={'dayOfWeek'} label={'Day of week'}/>
                     </div>
+                    <div css={tw`mt-6 bg-neutral-700 border border-neutral-800 shadow-inner p-4 rounded`}>
+                        <Switch
+                            name={'show_cheatsheet'}
+                            description={'Show the cron cheatsheet for some examples.'}
+                            label={'Show Cheatsheet'}
+                            defaultChecked={showCheatsheet}
+                            onChange={() => setShowCheetsheet(s => !s)}
+                        />
+                        {showCheatsheet &&
+                            <div css={tw`block md:flex w-full`}>
+                                <ScheduleCheatsheetCards/>
+                            </div>
+                        }
+                    </div>
                     <p css={tw`text-neutral-400 text-xs mt-2`}>
```

Comment:
The input values description should be above your component

Reply:
What do you mean by this?

Reply:
You've placed the component between the input description and the input boxes. Enabling the cheatsheet checkbox will further hide the description with its list